Logger/OutputLogger.php

```diff
@@ -33,24 +33,48 @@ class OutputLogger implements LoggerInterface
     const DEBUG  = 128;
     const ALL    = 255;
 
+    /**
+     * @var OutputInterface
+     */
     private $output;
+
+    /**
+     * @var int
+     */
     private $level = self::ALL;
 
+    /**
+     * OutputLogger constructor.
+     * @param OutputInterface $output
+     */
     public function __construct(OutputInterface $output)
     {
         $this->output = $output;
     }
 
+    /**
+     * @param $level
+     */
     public function setLevel($level)
     {
         $this->level = $level;
     }
 
+    /**
+     * @param string $message
+     * @param array $context
+     * @return void
```

Comment:
`@return void`? That is the same as `return null`. I believe this should be removed if we use PSR-2 + Symfony style.